debian/patches/do-not-use-upx.patch

```diff
@@ -0,0 +1,34 @@
+--- etcher-1.0.0~beta.15.orig/scripts/build/linux.sh
++++ etcher-1.0.0~beta.15/scripts/build/linux.sh
+@@ -32,11 +32,6 @@ if ! command -v bower 2>/dev/null; then
+   exit 1
+ fi
+ 
+-if ! command -v upx 2>/dev/null; then
+-  echo "Dependency missing: upx" 1>&2
+-  exit 1
+-fi
+-
+ if ! command -v python 2>/dev/null; then
+   echo "Dependency missing: python" 1>&2
+   exit 1
+@@ -102,11 +97,6 @@ function package_x86 {
+ 
+   mv $output_package/Etcher $output_package/etcher
+   chmod a+x $output_package/*.so*
+-
+-  # UPX fails for some reason with some other so libraries
+-  # other than libnode.so in the x86 build
+-  upx -9 $output_package/etcher $output_package/libnode.so
```

Comment:
The debian package tools automatically determine which shared library dependencies are needed by inspecting the binaries. If they are compressed with upx, then the debian tools cannot read them.

After I disabled this, it seems to me that the application starts a bit faster as well. This could just be my imagination though.